src/components/flag/flag.jsx

```diff
@@ -53,17 +54,17 @@ Flag.defaultProps = {
   round: false,
 };
 
-const unNest = (acc, i) => acc.concat(i);
-const dupInUpperCase = arr => arr.map(code => [code, code.toUpperCase()]).reduce(unNest, []);
-
-const countryCodes = dupInUpperCase(['ca', 'de', 'fr', 'ru', 'gb', 'dk', 'fi', 'no', 'se', 'us', 'jp', 'cn', 'eu']);
-const currencyCountryCodes = dupInUpperCase(['dk', 'no', 'se', 'us', 'eu']);
-
 Flag.propTypes = {
   className: PropTypes.string,
   /** A valid 2-character country code */
-  countryCode: PropTypes.oneOf(countryCodes),
-  secondaryCountryCode: PropTypes.oneOf(currencyCountryCodes),
+  countryCode: PropTypes.oneOf([
```

Comment:
I did this change so that the proptypes show up in styleguidist, sadly styleguidist does not work well with variables 😭

Before:
![image](https://cloud.githubusercontent.com/assets/3505504/25805930/d4cc4cf8-3401-11e7-8cfe-f8b78aa2ee9c.png)

After:
![image](https://cloud.githubusercontent.com/assets/3505504/25805888/a206a5a2-3401-11e7-95dc-7083fd2e7a48.png)